app/static/js/events/scheduler.js

```diff
@@ -859,10 +862,45 @@ function loadMicrolocationsToTimeline(day) {
         window.dayLevelTime.end.minutes = mainEvent.end_time.minutes();
     }
 
-    generateTimeUnits();
-
+    var least_hours = dayLevelTime.start.hours;
+    var least_minutes = dayLevelTime.start.minutes;
+    var max_hours = 0;
+    var max_minutes = 0;
     var dayIndex = _.indexOf(days, day);
 
+    if (/.+\/e\/.+\/schedule\//i.test(document.URL)) {
```

Comment:
@SaptakS use the `isReadOnly()` function. `isReadOnly()` returns a `true` if the scheduler is running the public page.